.github/workflows/ci.yaml

```diff
@@ -287,3 +287,47 @@ jobs:
             scripts/images/**/flatcar_production_*.sh
             scripts/images/**/flatcar_test_update.gz
             scripts/ebuild_logs.tar.xz
+  test:
+    name: "Run kola tests"
+    needs: packages
+    runs-on:
+      - debian
+      - build
+      - self-hosted
+      - x64
```

Comment:
I added ```amd64``` as a tag to the pool configured for kola tests. Instead of ```x64``` you can just add ```${{ matrix.arch }}```. This, together with the ```kola``` tag instead of the ```build``` tag, will make sure the tests are run on the proper architecture.
```suggestion
      - ${{ matrix.arch }}
```